### PR TITLE
DAOS-5180 test: Include HDF5 in soak

### PR DESCRIPTION
**Pull request by mjean308**
Test-tag-hw-large:pr,hw,large soak_smoke

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

### CODE REVIEW

**Review by shimizukko — APPROVED** [review]
LGTM

**Comment by johannlombardi** [comment]
@phender @saurabhtandan Could you please prioritize this review? This is a 1.1.1 blocker. Thanks!

File: src/tests/ftest/soak/soak_utils.py [hunk 1]
```diff
@@ -0,0 +1,606 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2019 Intel Corporation.
```
Comment (saurabhtandan) [inline src/tests/ftest/soak/soak_utils.py:3]: Minor: Should this be 2020?

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Pull request merged**